lib/idcl/ideucl-details.js

```diff
@@ -0,0 +1,186 @@
+const assignement = require('../shared/assignement');
+
+const nonNullIndexes = require('../shared/non-null-indexes');
+const trackDist = require('../shared/track-dist');
+
+/**
+* List the indexes that are truthy for both
+* @param {Array or number}
+* @param {Array or number}
+* @returns {number}
+*/
+
+const distancebox = function (box1, box2) {
+	let distance = 0;
+	if (typeof (box1) === 'number' && typeof (box2) === 'number') {
+		distance = Math.abs(box1 - box2);
+	} else if (box1 !== null && box2 !== null) {
+		const xDistance = (box1[0] - box2[0]) ** 2;
+		const yDistance = (box1[1] - box2[1]) ** 2;
+		distance = (xDistance + yDistance) ** 0.5;
+	}
+
+	return distance;
+};
+
+/**
+* List the indexes that are truthy for both
+* @param {Array}
```

Comment:
```suggestion
* @param {Array} trajectory
```

please fix it everywhere else